CTS_XML_TEI/perseus/latinLit/phi0959/.cts.xml

```diff
@@ -0,0 +1,8 @@
+<ti:textgroup
+  xmlns:ti="http://chs.harvard.edu/xmlns/cts"
+  urn="urn:cts:latinLit:phi0959"
+  tiid="annotsrc">
```

Comment:
hmm. are we going to have this specified in these files? Isn't the idea that they can be used in any aggregate inventory?